src/main/java/io/camunda/testing/utils/ProcessInstanceInspections.java

```diff
@@ -0,0 +1,100 @@
+package io.camunda.testing.utils;
+
+import io.camunda.testing.filters.ProcessInstanceRecordStreamFilter;
+import io.camunda.testing.utils.model.InspectedProcessInstance;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class ProcessInstanceInspections {
```

Comment:
:x: Please also add an example to the Readme like you did for "started by Timer"

(I like the new readme, by the way :trophy:)